package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "faceme",
+  "version": "1.0.0",
+  "description": "facial recognition",
+  "main": "app.js",
+  "dependencies": {
+    "aws-sdk": "^2.249.1",
+    "babel-polyfill": "^6.26.0",
+    "body-parser": "^1.18.2",
+    "crypto-js": "^3.1.9-1",
+    "dotenv": "^5.0.1",
+    "express": "^4.16.2",
+    "express-promise-router": "^3.0.2",
+    "fine-uploader": "^5.16.0",
+    "fine-uploader-wrappers": "^1.0.1",
+    "if-env": "^1.0.4",
+    "limiter": "^1.1.3",
+    "multiparty": "^4.1.3",
+    "pg": "^7.4.2",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2",
+    "react-fine-uploader": "^1.0.9",
+    "react-router-dom": "^4.2.2",
+    "react-transition-group": "^1.2.1",
+    "request-promise": "^4.2.2",
+    "rimraf": "^2.6.2"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "concurrently": "^3.5.0",
```

Comment:
prolly `npm install -g concurrently` and leave this out here, or add a script that utilizes this